extensions/wikidata/module/langs/translation-en.json

```diff
@@ -148,6 +148,10 @@
     "warnings-messages/multi-valued-property-is-required-for-new-item/body": "This property is expected to have more than one statement on each item but it has single statement, for instance on {example_entity}.",
     "warnings-messages/multi-valued-property-is-required-for-existing-item/title": "{property_entity} should have more than one statement on existing items.",
     "warnings-messages/multi-valued-property-is-required-for-existing-item/body": "This property is expected to have more than one statement on each item but it has single statement, for instance on {example_entity}. If the item already has statements with this property in Wikidata, then this warning can be ignored.",
+    "warnings-messages/no-references-found/title": "{property_entity} requires at least one reference.",
+    "warnings-messages/no-references-found/body": "Statements with property {property_entity} violating citation-needed constraint, it should have at least one reference",
```

Comment:
I think we can have only one issue type (whether the item is new or not should not matter, see below). The issue body is not very fluid in my opinion, how about this: `The property {property_entity} specifically requires at least one reference for its statements.`